src/browser/extension/options/index.js

```diff
@@ -1,79 +1,23 @@
 import React from 'react';
 import { render } from 'react-dom';
+import Options from './options';
 
 chrome.runtime.getBackgroundPage(background => {
   const syncOptions = background.syncOptions;
 
-  const saveOption = e => {
```

Comment:
I moved the `saveOption` logics from `index.js` to the concrete place where the options are saved (see [`MiscellaneousGroup.js:54`](https://github.com/zalmoxisus/redux-devtools-extension/pull/54/files#diff-3c2ffc337e1475e7b6bb59de0048a94bR18) for example). I believe it’s simpler and more flexible to let the options know how to store the data instead of writing a generic method that will guess this based on the control type.